cfe_internal/recommendations.cf

```diff
@@ -107,6 +107,46 @@ bundle agent ignore_interfaces_rx_reccomendations
 }
 @endif
 
+bundle agent nfs_mount_recommendations
+# @brief Recommendations about configured NFS servers
+{
+  meta:
+
+      "tags" slist => { "cfengine_recommends" };
+
+  vars:
+      "nfs_server_list"
+        slist => getvalues( "default:inventory_linux.nfs_server"),
+        depends_on => { "cfe_internal_inventory_mounted_nfs_server" };
+
+  classes:
+
+      # If we end up emitting the recommendation, then we define a class so that
+      # instructions about disabling these reports
```

Comment:
```
      # If we end up emitting the recommendation, then we define a class so that
      # instructions about disabling these reports are also emitted.
```